hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -386,7 +386,7 @@ public Response get(
       // http://localhost:9878/bucket/key?response-expires=1&response-expires=2
       // only response-expires=1 is valid
       MultivaluedMap<String, String> queryParams = context
-          .getUriInfo().getQueryParameters();
+          .getUriInfo().getQueryParameters(false);
```

Comment:
This looks fine, but I would like to know if we need a more thorough approach to handle Unicode values in the query params and how we process it.